src/uct/api/uct.h

```diff
@@ -773,18 +773,18 @@ void uct_worker_progress(uct_worker_h worker);
  * @ingroup UCT_CONTEXT
  * @brief Add a slow path callback function to a worker progress.
  *
- * If 'id_p' points to a NULL identifier, this function will add a callback which
- * will be called every time a progress is made on the worker.
+ * If 'id_p' points to UCS_CALLBACKQ_ID_NULL, this function will add a callback
+ * which will be called every time a progress is made on the worker.
```

Comment:
I suggest the following re-wording for clarity:

 If *id_p is equal to UCS_CALLBACKQ_ID_NULL, this function will add a callback
 which will be invoked every time progress is made on the worker. *id_p will
 be updated with an id which refers to this callback and can be used in
 @ref uct_worker_progress_unregister_safe to remove it from the progress path.